spec/operators/find-spec.ts

```diff
@@ -131,8 +131,9 @@ describe('find operator', () => {
     const source = hot('--a--b---c-|');
     const subs =       '^    !';
     const expected =   '-------(b|)';
+    const t =               '  |';
```

Comment:
nitpick: It's now technically valid for the times to be expressed with spaces, but they're easier to read with dashes. IMO, we should uses dashes here.

Reply:
I would agree, I'll revert to dashes. Anyway, I will convert these tests to run mode.